aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -67,18 +68,19 @@ def close(self):
         pass
 
     def save(self, data, ome_xml=None, channel_names=None, image_name="IMAGE0", pixels_physical_size=None,
-             channel_colors=None):
+             channel_colors=None, dimension_order="STZCYX"):
         """
         Save an image with the proper OME xml metadata.
 
         Parameters
         ----------
-        data: An array of dimensions TZCYX, ZCYX, or ZYX to be written out to a file.
+        data: An array of 3, 4, or 5 dimensions to be written out to a file.
```

Comment:
Is multiscene tiff writing not supported? Why not 6?

Reply:
Multiscene **ome** tiff is not supported. Multiscene tiff is fine. The OME spec doesn't support multiple scenes to my knowledge but really we should know this with the xslt stuff

Reply:
Let's figure this out tommorow, if the 4DN format supports it then include S. Doesn't make much sense to have to deal with multiple files for images of different parts of the same well.

Reply:
I believe ome does support multi scene but I did not want to add that functionality in this PR.  This PR is only for allowing users to pass dimension_order to the Writer, nothing else.